README.md

```diff
@@ -207,10 +207,14 @@ processors:
       separator: "::"
   queued_retry: {}
   tail_sampling:
-    policy:
-      name: policy1
-      type: rate_limiting
-      spans_per_second: 100
+    policies:
+      [
+        {
+          name: policy1,
+          type: rate_limiting,
+          rate_limiting: {spans_per_second: 100}
+        }
+      ]
```

Comment:
Generally we use the `- ` syntax for lists. Is there a reason to use JSON syntax?

Reply:
Used because I saw the json syntax being used in [tail_sampling_config.yaml](https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/samplingprocessor/tailsamplingprocessor/testdata/tail_sampling_config.yaml)

Will change to standard yaml syntax. :)